src/Storefront/Controller/StorefrontController.php

```diff
@@ -146,7 +147,11 @@ protected function forwardToRoute(string $routeName, array $attributes = [], arr
     {
         $router = $this->container->get('router');
 
-        $url = $this->generateUrl($routeName, $routeParameters, Router::PATH_INFO);
+        try {
+            $url = $this->generateUrl($routeName, $routeParameters, Router::PATH_INFO);
+        } catch (RouteNotFoundException) {
+            throw StorefrontException::routeNotFound($routeName);
```

Comment:
passing it as previous was correct, please add it back 😄 
```suggestion
        } catch (RouteNotFoundException $e) {
            throw StorefrontException::routeNotFound($routeName, $e);
```